packages/javascript/bh-shared-ui/src/components/UpdateUserForm/UpdateUserForm.tsx

```diff
@@ -124,38 +126,31 @@ const UpdateUserForm: React.FC<{
 
 const UpdateUserFormInner: React.FC<{
     onCancel: () => void;
-    onSubmit: (user: UpdatedUser) => void;
-    initialData: {
-        emailAddress: string;
-        principal: string;
-        firstName: string;
-        lastName: string;
-        SAMLProviderId?: string;
-        roles: number[];
-    };
+    onSubmit: (user: UpdateUserRequestForm) => void;
+    initialData: UpdateUserRequestForm;
     roles: any[];
-    SAMLProviders: any[];
+    SSOProviders?: SSOProvider[];
     isLoading: boolean;
     error: any;
-}> = ({ onCancel, onSubmit, initialData, roles, SAMLProviders, isLoading, error }) => {
+}> = ({ onCancel, onSubmit, initialData, roles, SSOProviders, isLoading, error }) => {
     const {
         control,
         handleSubmit,
         setValue,
         formState: { errors },
         watch,
-    } = useForm({
+    } = useForm<UpdateUserRequestForm & { authenticationMethod: 'sso' | 'password' }>({
         defaultValues: {
             ...initialData,
-            authenticationMethod: initialData.SAMLProviderId ? 'saml' : 'password',
+            authenticationMethod: initialData.SSOProviderId ? 'sso' : 'password',
         },
     });
 
     const authenticationMethod = watch('authenticationMethod');
 
     useEffect(() => {
         if (authenticationMethod === 'password') {
-            setValue('SAMLProviderId', '');
+            setValue('SSOProviderId', undefined);
```

Comment:
In create we set this as `''` but here we set it as `undefined` should these be matching? and if so which is the right value?

Reply:
Either should work since we check if the SSOProviderId exists.
Testing in the console:
```javascript
let thing = '';
if (!thing) console.log("is undefined");
```
Returns undefined.
I'll consolidate the `setValue` to `undefined`